src/test/resources/rego/perf/policy.rego

```diff
@@ -0,0 +1,5 @@
+package perf
+
+a := 42
+
+b := 1 + 2
```

Comment:
We should figure out some fun things to test here.

Reply:
`numbers.range` is a favorite :)